src/report/compiler.rs

```diff
@@ -208,4 +211,51 @@ mod tests {
         );
         std::env::remove_var("foundry_compilers_LOG");
     }
+
+    #[test]
+    fn fails_on_invalid_format() {
+        let result: Result<Target, _> = "invalid_format".parse();
+        assert!(result.is_err());
+    }
+    #[test]
+    fn fails_on_bad_name() {
+        let result: Result<Target, _> = "bad=in.json".parse();
```

Comment:
we can probably just get the err directly (as we know it's a bad input) with `.err().unwrap()`